styles/style.css

```diff
@@ -92,11 +96,37 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
+
+button {
+  background-color: #e95b1e;
+  color: white;
+  border-radius: 4px;
+  padding: 6px 6px;
+  border: 0px;
+}
 
+button:hover {
+  background-color: rgb(55, 178, 235);
+}
+.orange {
```

Comment:
instead of `.orange` can you use a css class name like `.btn-orange` do the same for `.white` in like 123>

Its a better naming convention